pages/add-user-page.tsx

```diff
@@ -90,103 +85,65 @@ export const AddUserPage = (props: NavigationComponentProps) => {
       <ScrollView
         contentInsetAdjustmentBehavior="automatic"
         style={backgroundStyle}>
-        <View
-          style={{
-            backgroundColor: isDarkMode ? Colors.black : Colors.white,
-          }}>
+        <View style={{ backgroundColor: isDarkMode ? Colors.black : Colors.white }}>
+
           <Title>Cadastrar Usuário</Title>
-          <Text style={Styles.inputTitle}> Nome </Text>
-          {nameError && <Text style={Styles.errorMessage}>
-            Insira um nome válido!
-          </Text>}
-          <TextInput 
-            value={name}
-            style={Styles.userInfoContainer} 
+
+          <FormField 
+            label='Nome'
+            validationError= {nameError}
+            validationMessage='Insira um nome válido!'
             onChangeText={(n) => setName(n)}
           />
-
-          <Text style={Styles.inputTitle}> E-mail </Text>
-          {emailError && <Text style={Styles.errorMessage}>
-            Insira um endereço de e-mail válido!
-          </Text>}
-          <TextInput 
-            value={email} 
-            keyboardType='email-address'
-            style={Styles.userInfoContainer} 
+          
+          <FormField 
+            label='E-mail'
+            validationError= {emailError}
+            validationMessage='Insira um endereço de e-mail válido!'
             onChangeText={(e) => setEmail(e)}
```

Comment:
```suggestion
            onChangeText={setEmail}
```

Vale pros outros campos tbm